### PR TITLE
ETS-2138 removes QS related files and variables

### PR DESCRIPTION
**Pull request by cllagosr**
⚠️⚠️⚠️ Please don't merge or deploy this version until the Monkeys decide to do so, we have a dependency here, we need to merge and deploy a ticket in QS before. ⚠️⚠️⚠️

### CODE REVIEW

File: source/scss/style.scss [hunk 1]
```diff
@@ -86,6 +86,3 @@
 
 // Form Checkbox
 @import "components/form-checkbox";
-
```
Comment (virgilchiriac) [inline source/scss/style.scss]: the checkbox was also added for QS. But if Planets want to keep it, ok...

**Review by virgilchiriac — DISMISSED** [review]

**Review by BirgitPohl — CHANGES_REQUESTED** [review]
It looks good. It needs a bit of cleanup, that was left in the past:

The remaining elements don't have proper states.

- I assume that the Atom - Loaders - Spinner is "complete"?
- I assume that Molecules - Tab-Content - Tab Content is "complete"?
- I assume that Molecules -  Form - Select is "complete"?
- I assume that Molecules - Form - Tag is "complete"?
- I assume that Molecules - Colorful-Tab  are "complete"?
- I assume that Atoms - Components - Close button  are "complete"? It should be moved to buttons, as it is embedded in a button-tag.
- I assume that Organisms - Colorful-Tab  are "complete"?
- I assume that Organisms - Tabs  are "complete"?
- The current calendar icon in the styleguide is in use by you and can be marked as "complete"?

Please, put the tabs in one folder.

File: source/scss/style.scss (hunk 1 above)
Comment (BirgitPohl) [reply source/scss/style.scss]: Happened with so many other elements. Checkboxes are a very general input element. Can be left here. It just needs a proper state. If this is is truly done, it should be marked as "complete" in the md files.

**Review by brunogeronimo — DISMISSED** [review]

**Review by MaikB-ts — DISMISSED** [review]

**Comment by cllagosr** [comment]
> It looks good. It needs a bit of cleanup, that was left in the past:
> 
> The remaining elements don't have proper states.
> 
> * I assume that the Atom - Loaders - Spinner is "complete"?
> * I assume that Molecules - Tab-Content - Tab Content is "complete"?
> * I assume that Molecules -  Form - Select is "complete"?
> * I assume that Molecules - Form - Tag is "complete"?
> * I assume that Molecules - Colorful-Tab  are "complete"?
> * I assume that Atoms - Components - Close button  are "complete"? It should be moved to buttons, as it is embedded in a button-tag.
> * I assume that Organisms - Colorful-Tab  are "complete"?
> * I assume that Organisms - Tabs  are "complete"?
> * The current calendar icon in the styleguide is in use by you and can be marked as "complete"?
> 
> Please, put the tabs in one folder.

I guess now the status of all these elements are up to your team, so if you think they are complete, I'll change them :)

**Comment by BirgitPohl** [comment]
> I guess now the status of all these elements are up to your team, so if you think they are complete, I'll change them :)

Ok, yes, please do so. :)

**Review by BirgitPohl — APPROVED** [review]

**Review by BirgitPohl — DISMISSED** [review]

**Review by virgilchiriac — DISMISSED** [review]

**Review by brunogeronimo — DISMISSED** [review]

**Review by brunogeronimo — APPROVED** [review]

**Pull request merged**